spec/features/remove_member_spec.rb

```diff
@@ -0,0 +1,130 @@
+require 'spec_helper'
+
+describe 'delete transaction', js: true do
```

Comment:
@yunixon also for speed-up this test, I would suggest you to add `js: true` only to contexts where you really need this. I see about half of contexts where you don't need js enabled.